deploy/crds/pravega-cr.yaml

```diff
@@ -0,0 +1,265 @@
+    #     timeoutSeconds: 5
+    #   livenessProbe:
+    #     initialDelaySeconds: 300
+    #     periodSeconds: 15
+    #     failureThreshold: 4
+    #     successThreshold: 1
+    #     timeoutSeconds: 5
+
+    # authImplementations:
+    #   mountPath:
+    #   authHandlers:
+    #   - image:
+    #     source:
+
+    # Pravega supports passing influxdb credentials as secret.
+    # It is the recommended approach rather than passing username/password as part of Pravega options.
+    # See https://github.com/pravega/pravega-operator/blob/master/doc/influxdb-auth.md
+
+    # influxDBSecret:
+    #   secret:
+    #   mountPath:
+
+    # DebugLogging indicates whether or not debug level logging is enabled.
+    # Defaults to false
+    debugLogging: false
+
+    longtermStorage:
+      filesystem:
+        persistentVolumeClaim:
+          claimName: pravega-tier2
+
+      # ecs:
+      #   configUri: http://10.247.10.52:9020?namespace=pravega
+      #   bucket: "shared"
+      #   prefix: "example"
+      #   credentials: ecs-credentials
+
+      # hdfs:
+      #   uri: hdfs://10.240.10.52:8020/
+      #   root: /example
+      #   replicationFactor: 3
+
+      # custom:
+      #   options:
+      #     pravegaservice.storage.layout: "CHUNKED_STORAGE"
+      #     pravegaservice.storage.impl.name: "S3"
+      #     s3.bucket: "aws-sdk-test"
+      #     s3.prefix: "10-11-1"
+      #     s3.connect.config.uri.override: "false"
+      #     s3.connect.config.uri: <uri>
+      #     s3.connect.config.access.key: <access key>
+      #     s3.connect.config.secret.key: <secret key>
+      #   env:
+      #     TIER2_STORAGE: "S3"
+      #     AWS_ACCESS_KEY_ID: "key"
+      #     AWS_SECRET_ACCESS_KEY: "secret"
+
+    # See https://github.com/pravega/pravega/blob/3f5b65084ae17e74c8ef8e6a40e78e61fa98737b/config/config.properties
+    # for available configuration properties
+    options:
+      bookkeeper.ensemble.size: "3"
+      bookkeeper.write.quorum.size: "3"
+      bookkeeper.ack.quorum.size: "3"
+      bookkeeper.write.timeout.milliseconds: "60000"
+      bookkeeper.write.outstanding.bytes.max: "33554432"
+      pravegaservice.cache.size.max: "1610612736"
+      pravegaservice.cache.time.seconds.max: "600"
+      pravegaservice.service.listener.port: "12345"
+      hdfs.block.size: "67108864"
+      writer.flush.threshold.bytes: "67108864"
+      writer.flush.size.bytes.max: "67108864"
+      pravegaservice.container.count: "8"
+      controller.container.count: "8"
+      controller.retention.bucket.count: "4"
+      controller.service.asyncTaskPool.size: "20"
+      controller.retention.thread.count: "4"
+      log.level: "INFO"
+      # emptyDirVolumeMounts: "heap-dump=/tmp/dumpfile/heap,logs=/opt/pravega/logs"
+      
+      # hostPathVolumeMounts: "foo=/tmp/foo,bar=/tmp/bar"
+      # configMapVolumeMounts: "prvg-logback:logback.xml=/opt/pravega/conf/logback.xml"
+
+      ## The mount dir for tls secrets is /etc/secret-volume
```

Comment:
Looks like tls and authentication section is duplicated again at lone no:244, probably you can remove one.